test/junit/scala/lang/stringinterpol/StringContextTest.scala

```diff
@@ -236,7 +236,7 @@ class StringContextTest {
       f" mind%n------%nmatter" ->
        """| mind
           |------
-          |matter""".stripMargin.linesIterator.mkString(compat.Platform.EOL),
+          |matter""".stripMargin.linesIterator.mkString(System.lineSeparator()),
```

Comment:
The extra parens aren't necessary, despite `scala.util.Properties.lineSeparator` calling it with parens.